django/forms/widgets.py

```diff
@@ -61,6 +63,64 @@ class MediaOrderConflictWarning(RuntimeWarning):
     pass
 
 
+@html_safe
+class MediaAsset:
+    element_template = "{path}"
+
+    def __init__(self, path, **attributes):
+        self._path = path
+        self.attributes = attributes
+
+    def __eq__(self, other):
+        return (self.__class__ is other.__class__ and self.path == other.path) or (
+            isinstance(other, str) and self._path == other
+        )
+
+    def __hash__(self):
+        return hash(self._path)
+
+    def __str__(self):
+        return format_html(
+            self.element_template,
+            path=self.path,
+            attributes=mark_safe(" ".join(self.escaped_attributes)),
```

Comment:
`flatatt` exists and also handles boolean attributes:
https://github.com/django/django/blob/042b381e2e37c0c37b8a8f6cc9947f1a2ebfa0dd/django/forms/utils.py#L20

I think reusing that [like here](https://github.com/matthiask/django-js-asset/blob/cd6be3edafb7864cb3bddf1608c039be25040c10/js_asset/js.py#L67) would make sense.

Reply:
Wonderful, thank you! I didn't know about this one.